### PR TITLE
Vb 2335 create active in active get session template end point fix

### PR DESCRIPTION
**Pull request by AledWEvans**
Bug detect my tom change how the code worked.

### CODE REVIEW

**Review by tpmcgowan — APPROVED** [review]
LGTM - tested locally and now works as expected 👍

**Comment by dhirajshettymoj** [comment]
should this be merged to develop?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt [hunk 1]
```diff
@@ -29,6 +29,32 @@ interface SessionTemplateRepository : JpaRepository<SessionTemplate, Long> {
...
+      "where u.prison.code = :prisonCode order by u.validFromDate,u.validToDate",
+  )
+  fun findSessionTemplatesByPrisonCode(
+    @Param("prisonCode") prisonCode: String,
+  ): List<SessionTemplate>
+
+  @Query(
+    "select u from SessionTemplate u " +
+      "where u.prison.code = :prisonCode " +
+      "and (u.validToDate >= CURRENT_DATE or u.validToDate is null) order by u.validToDate",
```
[the remark this answers is not in the document]
Comment (tpmcgowan) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/SessionTemplateRepository.kt:43]: yes, I tested that scenario, it counts as active and it does return those (which is what I think we need it to do) 👍

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**